frame/ddc-metrics-offchain-worker/src/lib.rs

```diff
@@ -283,11 +305,11 @@ impl<T: Trait> Module<T> {
         }
     }
 
-    fn get_start_of_day_ms() -> u64 {
-        let now = sp_io::offchain::timestamp();
-        let day_start_ms = (now.unix_millis() / MS_PER_DAY) * MS_PER_DAY;
-        day_start_ms
-    }
+    // fn get_start_of_day_ms() -> u64 {
```

Comment:
Please remove this code if we don't need it

Reply:
I will remove in next commit.